src/lib/features/playback-controls/ui/full-controls.svelte

```diff
@@ -32,21 +36,20 @@
       seek(e.detail.position);
     }}
   />
-  <div class="flex flex-row items-center justify-between">
-    <IconButton
-      name="Replay the last 10 seconds"
-      icon={IconReplay10}
-      class="w-9 h-9"
-      iconClass="w-7 h-7"
-      on:click={() => move(-10)}
-    />
-    <PlaybackButton prominent playing={!$paused} on:click={() => toggleGlobalPlayback()} />
-    <IconButton
-      name="Skip forward 30 seconds"
-      icon={IconForward30}
-      class="w-9 h-9"
-      iconClass="w-7 h-7"
-      on:click={() => move(30)}
-    />
-  </div>
+  <PlaybackControlButtons {prominent}>
+    <button
+      slot="pre"
+      name="Toggle playback speed"
+      class="flex items-center justify-center w-10 h-10 font-semibold text-sm rounded-full hover:bg-slate-600 active:bg-slate-500"
+      on:click={(e) => {
```

Comment:
suggestion (non-blocking): when adding event handlers to HTML nodes, we can make use of Svelte event modifiers like so:
```suggestion
      on:click|stopPropagation={() => {
```

Then we can remove the line from the handler.

question: why do we even need to stop propagation here? It was useful in the mini-player, but I don't think we need it here